src/app/pages/apps/store/available-apps-store.service.ts

```diff
@@ -0,0 +1,289 @@
+  readonly searchedApps$ = this.select((state) => state.searchedApps);
+  readonly appsByCategories$ = this.select((state) => state.appsByCategories);
+  readonly isLoading$ = this.select((state) => state.isLoading);
+  readonly appsByCategory$ = this.select((state) => state.appsByCategories);
+  readonly isFilterApplied$ = this.select((state) => state.isFilterApplied);
+  readonly searchQuery$ = this.select((state) => state.searchQuery);
+  readonly sliceAmount$ = this.select((state) => state.appsPerCategory);
+  readonly appsCategories$ = this.select((state) => state.categories);
+  readonly availableApps$ = this.select((state) => state.availableApps);
+  readonly selectedPool$ = this.select((state) => state.selectedPool);
+
+  constructor(
+    private dialogService: DialogService,
+    private errorHandler: ErrorHandlerService,
+    private appsService: ApplicationsService,
+    private translate: TranslateService,
+  ) {
+    super(initialState);
+    this.initialize();
+  }
+
+  readonly initialize = this.effect((triggers$: Observable<void>) => {
+    return triggers$.pipe(
+      tap(() => {
+        this.patchState({
+          ...initialState,
+          isLoading: true,
+        });
+      }),
+      switchMap(() => {
+        return combineLatest([
+          this.loadLatestApps(),
+          this.loadAvailableApps(),
+          this.loadCategories(),
+          this.loadAppsPool(),
+        ]);
+      }),
+      tap((
+        [
+          latestApps,
+          availableApps,
+          categories,
+          pool,
+        ]: [AvailableApp[], AvailableApp[], string[], string],
+      ) => {
+        this.patchState((state: AvailableAppsState): AvailableAppsState => {
+          const newState: AvailableAppsState = {
+            ...state,
+            latestApps: [...latestApps],
+            availableApps: [...availableApps],
+            categories: [...categories],
+            filteredApps: [],
+            recommendedApps: availableApps.filter((app) => app.recommended),
+            isLoading: false,
+            selectedPool: pool,
+          };
+          newState.appsByCategories = this.getAppsSlicesByCategory(newState);
+          return newState;
+        });
+      }),
+    );
+  });
+
+  resetFilters = this.updater((state: AvailableAppsState) => {
+    return {
+      ...state,
+      isFilterApplied: false,
+      searchQuery: '',
+      filteredApps: [],
+      searchedApps: [],
+    };
+  });
+
+  applyFilters(filter: AppsFiltersValues): void {
+    if (filter.categories.some((cat) => cat.includes(AppExtraCategory.NewAndUpdated))) {
+      filter.categories = filter.categories.filter((cat) => !cat.includes(AppExtraCategory.NewAndUpdated));
+      this.patchState((state) => {
+        return {
+          ...state,
+          isLoading: true,
+        };
+      });
+      this.appsService.getLatestApps(filter).pipe(
+        tap((filteredApps) => {
```

Comment:
IMO, it looks weird to put logic in `tap`, given that the `subscribe()` method is below. Also, it may be worth adding some conditions to destroy observables to avoid memory leaks.

I want to highlight that logic is similar for both cases. It may be worth consolidating it and reusing it.
`this.appsService.getLatestApps(filter)`
`this.appsService.getAvailableApps(filter)`